benchmarking/cluster_tpch_execution_times.txt

```diff
@@ -0,0 +1,27 @@
+Spark.executor.memory 6GB
+Spark.executor.cores 3
```

Comment:
Is spark running in docker here?

What's the input size? ~1GB?

Reply:
Yes, should i add it to the file?

Reply:
Yes add a note pointing to the dockerfiles used to generate the images

Also mention the size of the input

Reply:
if you just used the command`./dbgen` to generate the files, the total input is about 1GB across all tables

Reply:
Yes, I generated from dbgen. Added in file